app.js

```diff
@@ -0,0 +1,134 @@
+require('dotenv').config();
+
+let PORT = process.env.PORT;
+let express = require('express');
+let app = express();
+let db;
+const MongoClient = require('mongodb').MongoClient;
+const MONGO_URI = process.env.MONGO_URI_MLAB;
+const bodyParser = require('body-parser');
+
+MongoClient.connect(MONGO_URI, (err, database) => {
+  if(err) {
+    throw err;
+  }
+
+  db = database;
+
+  app.listen(PORT, () => {
+    console.log(`Server is up and running on port ${PORT}`);
+  });
+});
+
+app.use(bodyParser.json());
+
+app.get('/initializeDatabase', (req, res) => {
+  const users = [
+    { "userId": 1, "firstName": "Amr", "lastName": "Desouky" },
+    { "userId": 2, "firstName": "Ernie", "lastName": "Casilla" },
+    { "userId": 3, "firstName": "Ben", "lastName": "Sammons" } ];
+  const accounts = [
+    { "type": "Checking", "userId": 1, "amount": 300.00, "currency": "USD", "timestamp": Date.now() },
+    { "type": "Saving",   "userId": 1, "amount": 200.00, "currency": "USD", "timestamp": Date.now() },
+    { "type": "Checking", "userId": 2, "amount": 300.00, "currency": "EUR", "timestamp": Date.now() },
+    { "type": "Saving",   "userId": 3, "amount": 300.00, "currency": "USD", "timestamp": Date.now() } ];
+
+  db.collection('users').remove((error, result) => { });
+  db.collection('accounts').remove((error, result) => { });
+  db.collection('users').insert(users, (error, result) => { });
+  db.collection('accounts').insert(accounts, (error, result) => { });
+
+  res.send('done');
+});
+
+app.get('/account/:userID/balance', (req, res) => {
+  const userId = parseInt(req.params.userID);
```

Comment:
You might want to set a radix and do a isNan guard check just in case the user sends a incorrect input

Reply:
Thanks for your input! I will be going back through the code and adding these types of checks and remove my TODOs regarding incorrect input and error handling